whitelist-authenticator-schools/src/main/java/de/intension/authentication/schools/SchoolWhitelistAuthenticatorFactory.java

```diff
@@ -84,24 +89,33 @@ public List<ProviderConfigProperty> getConfigProperties()
                                "User attribute which contains the school ids",
                                ProviderConfigProperty.STRING_TYPE, USER_ATTRIBUTE_PARAM_DEFAULT),
                        new ProviderConfigProperty(AUTH_WHITELIST_REALM, "Realm",
-                                                  "Specifies the name of the realm that contains the configured client "
-                                                      + "for the REST API. If no value is specified, then the current realm is used.",
-                                                  ProviderConfigProperty.STRING_TYPE, null),
+                               "Specifies the name of the realm that contains the configured client "
+                                       + "for the REST API. If no value is specified, then the current realm is used.",
+                               ProviderConfigProperty.STRING_TYPE, null),
                        new ProviderConfigProperty(AUTH_WHITELIST_CLIENT_ID, "Client ID",
-                                                  "REST-API Client ID",
-                                                  ProviderConfigProperty.STRING_TYPE, null),
+                               "REST-API Client ID",
+                               ProviderConfigProperty.STRING_TYPE, null),
+                       new ProviderConfigProperty(AUTH_WHITELIST_CLIENT_GRANT_TYPE, "OAuth Grant Type",
```

Comment:
Maybe we should use a drop-down for the "OAuth Grant Type" instead of a free-text